### DISCUSSION TITLE
Welcome to shadowsocks-android Discussions!

### DISCUSSION THREAD

**Discussion by madeye** [shadowsocks/shadowsocks-android#2616] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Use English!
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ariank47** [comment]
Hey!
I've recently switched to your app from Clash4android. I faced some issues with opening youtube there. Other than that there was a nice factor in that app which you were able to select a ''SELECTOR'' to connect you to the fastest URL. Is there a similar option in your app too. I would love to know because I have a ton of ports and I can't test them all manually!
Regards
